en/advanced_features/rtk-gps.md

```diff
@@ -1,98 +1,135 @@
 # RTK GPS
 
-RTK (Real Time Kinematik) increases the GPS accuracy to a
-centimeter-level. PX4 supports RTK GPS using the 
-[u-blox M8P](https://www.u-blox.com/en/product/neo-m8p) 
-GPS device. This page describes the required setup.
+[Real Time Kinematic (RTK)](https://en.wikipedia.org/wiki/Real_Time_Kinematic) increases the accuracy of GNSS/GPS systems to centimeter-level. RTK allows PX4 to be used in applications like precision surveying, where pinpoint accuracy is essential.

```

Comment:
I have put "precision surveying" here. If you have any other better/good examples I could put in that would be cool.